docs/pages/jpscore/jpscore_inifile.md

```diff
@@ -6,267 +6,1219 @@ sidebar: jupedsim_sidebar
+
+and **closed** at 20, 70 and 100.
+
+{% include note.html content="all times are in seconds."%}
+
+# Agents
+
+There are two ways to distribute agents for a simulation:
+
+- [random distribution](#agents_distribution) in a specific area *before* the simulation starts.
+- distribution by means of [sources](#sources) *during* the simulation.
+
+```xml
+
+<agents>
+    <agents_distribution>
+        <group group_id="1" room_id="0" number="10"/>
+        <group group_id="2" room_id="0" subroom_id="0" number="10" goal_id="" router_id="1"/>
+    </agents_distribution>
+    <agents_sources>
+        <source id="1" frequency="2" agents_max="10" group_id="1" caption="caption" greedy="true"/>
+        <source id="2" time="10" agent_id="50" group_id="1" caption="caption" greedy="true"/>
+        <source id="10" caption="new-source" time_min="5" time_max="30" frequency="5" N_create="10" agents_max="300"
+                group_id="0" x_min="0" x_max="3" y_min="0" y_max="3" percent="0.5" rate="2" greedy="true"/>
+        <file>sources.xml</file>
+    </agents_sources>
+</agents>
+```
+
+## Agents_distribution
+
+Above is an example how to define agent's characteristics with different number of attributes, which are as follows:
+
+- `group_id`: mandatory parameter defining the unique id of that group.
+
+- `number`: mandatory parameter defining the number of agents to distribute.
+
+- `room_id`: mandatory parameter defining the room where the agents should be randomly distributed.
+
+- `subroom_id`: defines the id of the subroom where the agents should be distributed. If omitted then the agents are
+  homogeneously distributed in the room.
+
+- `goal_id`: should be one of the `id`s defined in the section  [goals](#goals). If omitted or is `-1` then the shortest
+  exit to the outside is chosen by the agent.
+
+- `router_id`: defines the route choice model to be used. See [routing documentation](#routing) of available routers.
+
+- `agent_parameter_id`: choose a set of parameters for the [operational models](#operational-models).
+
+- `x_min`, `x_max`, `y_min` and `y_max`: define a bounding box where agents should be distributed.
+
+- `startX`, `startY`: define the initial coordinate of the agents. This might be useful for testing/debugging. Note that
+  these two options are only considered if `number=1`.
+
+- `pre_movement_mean` and `pre_movement_sigma`: premovement time is Gauss-distributed $$\mathcal{N}(\mu, \sigma^2)$$.
+
+## Sources
+
+Besides distributing agents randomly before the simulation starts, it is possible to define sources in order to "inject"
+new agents in the system during the simulation.
+
+An example of usage:
+
+- Busses are coming every 10 min (600 seconds).
+- Every bus transports 100 pedestrians.
+- When the bus stops, every 2 seconds 10 pedestrians leave the bus.
+- 3 Buses at max.
+
+```xml
+
+<source id="10" frequency="600" N_create="100" agents_max="300"
+        percent="0.1" rate="2"/>
+```
+
+### Parameters
```

Comment:
Since those are XML attributes I would prefer if this section is named `### Source attributes`